frontend/app/.server/routes/mappers/benefit-renewal.state.mapper.ts

```diff
@@ -264,6 +278,53 @@ export class DefaultBenefitRenewalStateMapper implements BenefitRenewalStateMapp
     };
   }
 
+  mapRenewChildStateToChildBenefitRenewalDto({ applicationYear, addressInformation, children, clientApplication, contactInformation, hasAddressChanged, hasMaritalStatusChanged, maritalStatus, partnerInformation }: RenewChildState): ChildBenefitRenewalDto {
+    const hasEmailChanged = contactInformation.isNewOrUpdatedEmail;
+    if (hasEmailChanged === undefined) {
+      throw Error('Expected hasEmailChanged to be defined');
+    }
+
+    const hasPhoneChanged = contactInformation.isNewOrUpdatedPhoneNumber;
+    if (hasPhoneChanged === undefined) {
+      throw Error('Expected hasPhoneChanged to be defined');
+    }
+
+    return {
+      ...clientApplication,
+      applicantInformation: this.toApplicantInformation({
+        existingApplicantInformation: clientApplication.applicantInformation,
+        hasMaritalStatusChanged,
+        renewedMaritalStatus: maritalStatus,
+      }),
+      applicationYearId: applicationYear.id,
+      children: this.toChildren({
+        existingChildren: clientApplication.children,
+        renewedChildren: children,
+      }),
+      changeIndicators: {
+        hasAddressChanged,
+        hasEmailChanged,
+        hasMaritalStatusChanged,
+        hasPhoneChanged,
+      },
+      contactInformation: this.toContactInformation({
+        renewedAddressInformation: addressInformation,
+        renewedContactInformation: contactInformation,
+        existingContactInformation: clientApplication.contactInformation,
+        hasAddressChanged,
+        hasEmailChanged,
+        hasPhoneChanged,
+      }),
+      partnerInformation: this.toPartnerInformation({
+        existingPartnerInformation: clientApplication.partnerInformation,
+        hasMaritalStatusChanged: true,
```

Comment:
Since we should be using result of `hasMaritalStatusChanged`

```suggestion
        hasMaritalStatusChanged,
```